benchmarks/benchmark-solve-cra.C

```diff
@@ -49,18 +49,19 @@ using Field = Givaro::ZRing<Integer>;
 #endif
 
 template <class Field, class Matrix>
-static bool checkResult(const Field& ZZ, Matrix& A, BlasVector<Field>& B, BlasVector<Field>& X, Integer& d)
+static bool checkResult(Matrix& A, BlasVector<Field>& B, BlasVector<Field>& X, Integer& d)
 {
+    Field ZZ;
     BlasVector<Field> B2(ZZ, A.coldim());
     BlasVector<Field> B3(ZZ, A.coldim());
     A.apply(B2, X);
-
-    Integer tmp;
+   
     for (size_t j = 0; j < B.size(); ++j) {
-        B3.setEntry(j, d * B.getEntry(j));
+        ZZ.mul(B3[j],B[j],d);
     }
+
     for (size_t j = 0; j < A.coldim(); ++j) {
-        if (!ZZ.areEqual(B2[j], B3[j])) {
+        if (!ZZ.isZero(B2[j]-B3[j])) {
```

Comment:
This introduces a mistake: don't do inline operators, but ZZ.sub instead. In this case, the ZZ.areEqual was better, and even better would be to write VectorDomain::areEqual(B2,B3);

Reply:
I fixed that in my commits. You should review [final changes](https://github.com/linbox-team/linbox/pull/147/files) directly.

Reply:
Oups, sorry, I did not notice that there was a later commit by you.
All good then. Still testing and then merging.